src/main/java/org/jvnet/hudson/test/JenkinsHomeLoader.java

```diff
@@ -36,11 +36,11 @@
 import javax.annotation.CheckForNull;
 
 /**
- * Controls how a {@link HudsonTestCase} initializes <tt>JENKINS_HOME</tt>.
+ * Controls how a {@link JenkinsTestCase} initializes <tt>JENKINS_HOME</tt>.
  *
  * @author Kohsuke Kawaguchi
  */
-public interface HudsonHomeLoader {
+public interface JenkinsHomeLoader {
```

Comment:
No, this is incompatible: https://github.com/search?q=user%3Ajenkinsci+HudsonHomeLoader&type=Code